test/mocha.opts

```diff
@@ -1,4 +0,0 @@
---ui tdd
---timeout 30s
---reporter spec
---require source-map-support/register
```

Comment:
Looks like sourcemaps work fine from tests using the current configuration, so 👍.